### PR TITLE
Add TravelPlan Package with TravelPlan and Wishlist Class

### PR DESCRIPTION
**Pull request by jeannetoh99**

### CODE REVIEW

File: src/main/java/seedu/address/model/travelplan/ActivityList.java [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+    public void resetData(ReadOnlyActivityList newData) {
+        requireNonNull(newData);
+
+        setActivities(newData.getActivityList());
+    }
+
+    //// person-level operations
+
+    /**
+     * Returns true if a person with the same identity as {@code person} exists in the address book.
```
Comment (jiaweiteo) [inline src/main/java/seedu/address/model/travelplan/ActivityList.java:62]: Should change person to activity

File: src/main/java/seedu/address/model/travelplan/ActivityList.java [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+     */
+    public boolean hasActivity(Activity activity) {
+        requireNonNull(activity);
+        return activities.contains(activity);
+    }
+
+    /**
+     * Adds a person to the address book.
```
Comment (jiaweiteo) [inline src/main/java/seedu/address/model/travelplan/ActivityList.java:70]: Same

File: src/main/java/seedu/address/model/wishlist/Wishlist.java [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+     *   among constructors.
+     */
+    {
+        activities = new UniqueActivityList();
+    }
+
+    public Wishlist() {}
+
+    /**
+     * Creates an AddressBook using the Activitys in the {@code toBeCopied}
```
Comment (jiaweiteo) [inline src/main/java/seedu/address/model/wishlist/Wishlist.java:33]: Wishlist

File: src/main/java/seedu/address/model/wishlist/Wishlist.java [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+     */
+    public Wishlist(ReadOnlyActivityList toBeCopied) {
+        this();
+        resetData(toBeCopied);
+    }
+
+    //// list overwrite operations
+
+    /**
+     * Replaces the contents of the person list with {@code activities}.
```
Comment (jiaweiteo) [inline src/main/java/seedu/address/model/wishlist/Wishlist.java:43]: activities

**Review by jiaweiteo — CHANGES_REQUESTED** [review]
Just some javadocs comments to fix for most of the classes

File: src/main/java/seedu/address/model/wishlist/Wishlist.java (hunk 4 above)
Comment (timjkong) [inline src/main/java/seedu/address/model/wishlist/Wishlist.java:43]: change person to activity

File: src/main/java/seedu/address/model/wishlist/Wishlist.java [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+     * Returns true if a person with the same identity as {@code person} exists in the address book.
+     */
+    public boolean hasActivity(Activity activity) {
+        requireNonNull(activity);
+        return activities.contains(activity);
+    }
+
+    /**
+     * Adds a person to the address book.
+     * The person must not already exist in the address book.
```
Comment (timjkong) [inline src/main/java/seedu/address/model/wishlist/Wishlist.java:71]: person change to activity, addressbook change to wishlist

File: src/main/java/seedu/address/model/wishlist/Wishlist.java [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+     */
+    public void setActivity(Activity target, Activity editedActivity) {
+        requireNonNull(editedActivity);
+
+        activities.setActivity(target, editedActivity);
+    }
+
+    /**
+     * Removes {@code key} from this {@code AddressBook}.
+     * {@code key} must exist in the address book.
```
Comment (timjkong) [inline src/main/java/seedu/address/model/wishlist/Wishlist.java:90]: addressbook change to wishlist

File: src/main/java/seedu/address/model/AddressBook.java [hunk 7]
```diff
@@ -17,7 +17,7 @@
     private final UniquePersonList persons;
 
     /*
-     * The 'unusual' code block below is a non-static initialization block, sometimes used to avoid duplication
+     * The 'unuasual' code block below is a non-static initialization block, sometimes used to avoid duplication
```
Comment (timjkong) [inline src/main/java/seedu/address/model/AddressBook.java:20]: think you accidentally added 'a'?

**Review by timjkong — COMMENTED** [review]
looks good :-) just update the javadocs?

**Comment by lyeyixian** [comment]
Don't merge first after commiting changes. 
I need to think about all the ReadOnlyXXX classes, if they are needed

**Review by jiaweiteo — APPROVED** [review]
LGTM

**Pull request merged**